features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -1095,11 +1099,11 @@
   "console.analytics.scopes.update": ["internal_governance_update"],
   "console.analytics.scopes.read": ["internal_governance_view"],
   "console.api_resources.enabled": true,
-  "console.api_resources.scopes.feature": [],
-  "console.api_resources.scopes.create": ["internal_application_mgt_create"],
-  "console.api_resources.scopes.read": ["internal_application_mgt_view"],
-  "console.api_resources.scopes.update": ["internal_application_mgt_update"],
-  "console.api_resources.scopes.delete": ["internal_application_mgt_update"],
+  "console.api_resources.scopes.feature": ["console:apiResources"],
+  "console.api_resources.scopes.create": ["internal_api_resource_create"],
```

Comment:
There is a possibility of extension logics might break due to this. Check those possibilities as well.

Reply:
Sure, will add the required changes for the extensions side.